src/main/java/com/gizasystems/automationexercise/pages/ProductQuantity.java

```diff
@@ -0,0 +1,70 @@
+package com.gizasystems.automationexercise.pages;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+
+public class ProductQuantity {
```

Comment:
Please add "Page" at the end of the page class name.

Reply:
Solved.